APIScrape/README.md

```diff
@@ -0,0 +1,70 @@
+TheCatApi // Developer Experience
```

Comment:
This reamde doesn't tell me how to do the API Scrape. It tells me how to use someone else's API.
I need documentation (instructions) for using YOUR code.

Reply:
The README that tells you how to do the API Scrape is in the root folder.

Reply:
I'll do this